low-level/framework/utils/store_queue.py

```diff
@@ -92,8 +99,11 @@ def _create_space(self, size_of_item, reclaimed_space=0):
     def get(self):
         if self.is_empty():
             return
-        item = store.get(f"SSPL_UNSENT_MESSAGES/{self.head}")
-        store.delete(f"SSPL_UNSENT_MESSAGES/{self.head}")
+        # item = store.get(f"SSPL_UNSENT_MESSAGES/{self.head}")
+        # store.delete(f"SSPL_UNSENT_MESSAGES/{self.head}")
+        SSPL_UNSENT_MESSAGES = os.path.join(self.cache_dir_path, f"SSPL_UNSENT_MESSAGES_{self.head}")
+        item = store.get(SSPL_UNSENT_MESSAGES)
```

Comment:
Retrieving all the messages from queue, without using head? 
@sandeepsgt this would break the logic !